src/Accounts/Accounts/AzureRmAlias/Mappings.json

```diff
@@ -840,7 +840,8 @@
     "New-AzDataMigrationSelectedDB": "New-AzureRmDataMigrationSelectedDB",
     "New-AzDataMigrationSyncSelectedDB": "New-AzureRmDataMigrationSyncSelectedDB",
     "New-AzDmsSelectedDBObject": "New-AzureRmDmsSelectedDBObject",
-    "New-AzDmsSyncSelectedDBObject": "New-AzureRmDmsSyncSelectedDBObject"
+    "New-AzDmsSyncSelectedDBObject": "New-AzureRmDmsSyncSelectedDBObject",
```

Comment:
We are not adding mappings for new cmdlets, this is strcitly for compatibility with scripts written against AzureRM